spec/features/users_spec.rb

```diff
@@ -140,4 +139,121 @@
+        scenario '表示される内容が正しいこと（フォームの内容やボタン、リンク等が正しく表示されていること）' do
+          sign_in @user
+          visit(edit_user_path(@user))
+          expect(page).to have_field 'お名前', with: @user.profile.name
+          expect(page).to have_field 'ご住所', with: @user.profile.address
+          expect(page).to have_field 'お電話番号', with: @user.profile.phone_number
+
+          expect(page).to have_field '生年月日', with: @user.profile.birthday.strftime("%Y-%m-%d")
+          expect(page).to have_field '飼主経験', with: @user.profile.breeding_experience
+          expect(page).to have_button 'プロフィール内容変更'
+          expect(page).to have_field 'メールアドレス', with: @user.email
+          expect(page).to have_button 'アカウント情報更新'
+        end
+        scenario 'プロフィール正しく値を入力した場合、ユーザーのマイページ画面に遷移すること' do
+          sign_in @user
+          visit(edit_user_path(@user))
+          fill_in 'お名前', with: 'KAZUYA'
+          fill_in 'ご住所', with: '大阪市'
+          fill_in 'お電話番号', with: '00000000000'
+          fill_in '生年月日', with: '2022-06-26'
+          fill_in '飼主経験', with: '猫1年'
+          click_button 'プロフィール内容変更'
+          expect(page).to have_current_path user_path(@user)
+        end
+      end
+
+      context '異常系' do
+        scenario 'プロフィール正しく値を入力なかった場合、バリデーションエラーの内容が表示されること' do
+          sign_in @user
+          visit(edit_user_path(@user))
+          fill_in 'お名前', with: nil
+          fill_in 'ご住所', with: nil
+          fill_in 'お電話番号', with: nil
+          fill_in '生年月日', with: nil
+          fill_in '飼主経験', with: nil
+          click_button 'プロフィール内容変更'
+          expect(page).to have_content 'お名前 入力されていません。'
+          expect(page).to have_content 'ご住所 入力されていません。'
+          expect(page).to have_content 'お電話番号 入力されていません。'
+          expect(page).to have_content 'お電話番号 が無効です。'
+          expect(page).to have_content '生年月日 入力されていません。'
+          expect(page).to have_content '飼主経験 入力されていません。'
+        end
+      end
+    end
+  end
+  describe 'アカウント情報' do
+    before do
+      @user = create(:user, email: 'test123456789@test.com', password: 'password', password_confirmation: 'password')
+      @user.confirm
+    end
+    context '正常系' do
+      scenario 'アカウント情報を正しく入力した場合、Home 画面に遷移すること' do
+        sign_in @user
+        visit(edit_user_path(@user))
+        fill_in 'メールアドレス', with: 'test123456789@test.com'
+        fill_in '現在のパスワード', with: 'password'
+        fill_in 'パスワード', with: 'password123'
+        fill_in 'パスワード（確認用）', with: 'password123'
+        click_button 'アカウント情報更新'
+        expect(page).to have_current_path root_path
+      end
+
+      scenario 'アカウント情報を正しく入力した場合、flash メッセージが正しく表示されること' do
+        sign_in @user
+        visit(edit_user_path(@user))
+        fill_in 'メールアドレス', with: 'test123456789@test.com'
+        fill_in '現在のパスワード', with: 'password'
+        fill_in 'パスワード', with: 'password123'
+        fill_in 'パスワード（確認用）', with: 'password123'
+        click_button 'アカウント情報更新'
+        expect(page).to have_content 'アカウント情報を変更しました。'
+      end
```

Comment:
この辺をなるべく共通化して書くと以下のような感じになります。
よければ参考にしてください。

```suggestion
  describe 'アカウント情報' do
    let(:user) do
      create(:user, email: 'test123456789@test.com', password: 'password', password_confirmation: 'password', &:confirm)
    end

    before do
      sign_in user
      visit edit_user_path(user)
    end

    context '正常系' do
      before do
        fill_in 'メールアドレス', with: 'test123456789@test.com'
        fill_in '現在のパスワード', with: 'password'
        fill_in 'パスワード', with: 'password123'
        fill_in 'パスワード（確認用）', with: 'password123'
        click_button 'アカウント情報更新'
      end

      scenario 'アカウント情報を正しく入力した場合、Home 画面に遷移すること' do
        expect(page).to have_current_path root_path
      end

      scenario 'アカウント情報を正しく入力した場合、flash メッセージが正しく表示されること' do
        expect(page).to have_content 'アカウント情報を変更しました。'
      end
    end
```